cmd/internal/cli/pull.go

```diff
@@ -170,6 +181,38 @@ func pullRun(cmd *cobra.Command, args []string) {
 		if err != nil {
 			sylog.Fatalf("%v\n", err)
 		}
+
+		// check if we pulled from the library, if so; is it signed?
+		if PullLibraryURI != "" && !unauthenticatedPull {
```

Comment:
The `PullLibraryURI` is to check if we pulled from the library, as opposed to docker or shub. Because container pulled from docker/shub this way will not be signed, so we might as well skip the verifying.